cfme/fixtures/candu.py

```diff
@@ -90,3 +123,367 @@ def candu_db_restore(temp_appliance_extended_db):
     app.db.fix_auth_dbyml()
     app.evmserverd.start()
     app.wait_for_web_ui()
+
+
+@pytest.fixture(scope="module")
+def vm_ownership(enable_candu, provider, appliance):
+    # Chargeback/metering report tests will filter on VM owner,
+    # so we assign the chargeback VM's owner here.
+    vm_name = provider.data['cap_and_util']['chargeback_vm']
+    vm = appliance.provider_based_collection(provider, coll_type='vms').instantiate(vm_name,
+                                                                                    provider)
+    if not vm.exists_on_provider:
+        pytest.skip(f'Skipping test: {vm_name} VM does not exist')
+    vm.mgmt.ensure_state(VmState.RUNNING)
+
+    group_collection = appliance.collections.groups
+    cb_group = group_collection.instantiate(description='EvmGroup-user')
+
+    user = None
+    try:
+        user = appliance.collections.users.create(
+            name=f'{provider.name}_{fauxfactory.gen_alphanumeric()}',
```

Comment:
`fauxfactory.gen_alphanumeric` (and its other methods) take a `start` kwarg that allows you to define the start of the random string - please use it instead of f-strings here.

`fauxfactory.gen_alphanumeric(start=f'{provider.name_})`  <- here you still kinda need an f-string if you want to have an underscore there